AsyncDisplayKitTests/ASVideoNodeTests.m

```diff
@@ -42,103 +43,151 @@ @implementation ASVideoNodeTests
 
 - (void)setUp
 {
-  _videoNode = [[ASVideoNode alloc] init];
+  // _videoNode = [[ASVideoNode alloc] init];
   _firstAsset = [AVURLAsset assetWithURL:[NSURL URLWithString:@"firstURL"]];
   _secondAsset = [AVAsset assetWithURL:[NSURL URLWithString:@"secondURL"]];
+  _url = [NSURL URLWithString:@"testURL"];
 }
 
-- (void)testVideoNodeReplacesAVPlayerItemWhenNewURLIsSet
+
+- (void)testSpinnerDefaultsToNil
 {
-  _videoNode.interfaceState = ASInterfaceStateFetchData;
-  _videoNode.asset = _firstAsset;
-  
-  AVPlayerItem *item = [_videoNode currentItem];
-  
-  _videoNode.asset = _secondAsset;
-  AVPlayerItem *secondItem = [_videoNode currentItem];
-  
-  XCTAssertNotEqualObjects(item, secondItem);
+  XCTAssertNil(_videoNode.spinner);
 }
 
-- (void)testVideoNodeDoesNotReplaceAVPlayerItemWhenSameURLIsSet
-{
-  _videoNode.interfaceState = ASInterfaceStateFetchData;
 
-  _videoNode.asset = _firstAsset;
-  AVPlayerItem *item = [_videoNode currentItem];
-  
-  _videoNode.asset = [AVAsset assetWithURL:_firstAsset.URL];
-  AVPlayerItem *secondItem = [_videoNode currentItem];
-  
-  XCTAssertEqualObjects(item, secondItem);
+- (void)testOnPlayIfVideoIsNotReadyInitializeSpinnerAndAddAsSubnode
+{
+  _videoNode = [[ASVideoNode alloc] initWithAsset:_firstAsset];
+  [self doOnPlayIfVideoIsNotReadyInitializeSpinnerAndAddAsSubnodeWithUrl];
 }
 
-- (void)testSpinnerDefaultsToNil
+- (void)testOnPlayIfVideoIsNotReadyInitializeSpinnerAndAddAsSubnodeWithUrl
```

Comment:
Thanks for taking a look at the tests and making them more descriptive!  If we don't want the init in the method above, then just delete that line.